Modules/@babylonjs/react-native/package.json

```diff
@@ -43,7 +43,7 @@
     "@types/jest": "^25.2.1",
     "@types/react": "^16.9.32",
     "@types/react-native": "^0.63.1",
-    "@types/react-native-permissions": "^2.0.0",
+    "react-native-permissions": "^4.1.5",
```

Comment:
Seems like we shouldn't need this if the types are already included in the package. Or were they only included as of 4.0?

Reply:
So for some reason when running in new architecture, the `@babylonjs/react-native` package needs the implementation of `react-native-permissions` in it's corresponding `node_modules` folder without it the mentioned error gets thrown. Also types are included as of 3+

Reply:
It looks like it's a standard practice to add your peer dependencies as dev dependencies in development. 

Here is an example of react native builder bob (popular create-react-native-library) which adds `react-native` as dev dependency: https://github.com/callstack/react-native-builder-bob/blob/9a025b2a89c7790b74862bf95ccab4b6a388e2cb/packages/create-react-native-library/templates/common/%24package.json#L90